src/pages/workspaces/List.js

```diff
@@ -282,15 +273,61 @@ export const WorkspaceList = () => {
                   side: 'left'
                 }, [
                   Utils.switchCase(lastRunStatus,
-                    ['success', () => icon('success-standard', { size: 20, style: { color: colors.success() } })],
-                    ['failure', () => icon('error-standard', { size: 20, style: { color: colors.danger(0.85) } })],
-                    ['running', () => icon('sync', { size: 20, style: { color: colors.success() } })]
+                    ['success', () => icon('success-standard', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Success'
+                    })],
+                    ['failure', () => icon('error-standard', {
+                      size: 20, style: { color: colors.danger(0.85) },
+                      'aria-label': 'Workflow Status Failure'
+                    })],
+                    ['running', () => icon('sync', {
+                      size: 20, style: { color: colors.success() },
+                      'aria-label': 'Workflow Status Running'
+                    })]
                   )
                 ])
               ])
             ])
           },
           size: { basis: 30, grow: 0, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }),
+          cellRenderer: ({ rowIndex }) => {
+            const { workspace: { cloudPlatform } } = sortedWorkspaces[rowIndex]
+            return div({ style: { ...styles.tableCellContainer, paddingRight: 0 } }, [
+              div({ style: styles.tableCellContent }, [
+                Utils.switchCase(cloudPlatform,
+                  ['Gcp', () => h(CloudGcpLogo, { title: 'Google Cloud Platform', role: 'img' })],
+                  ['Azure', () => h(CloudAzureLogo, { title: 'Microsoft Azure', role: 'img' })])
+              ])
+            ])
+          },
+          size: { basis: 30, grow: 0, shrink: 0 }
+        }, {
+          headerRenderer: () => div({ className: 'sr-only' }, ['Actions']),
+          cellRenderer: ({ rowIndex }) => {
+            const { accessLevel, workspace: { workspaceId, namespace, name } } = sortedWorkspaces[rowIndex]
+            if (!Utils.canRead(accessLevel)) {
+              // No menu shown if user does not have read access.
+              return null
```

Comment:
Similar to https://github.com/DataBiosphere/terra-ui/pull/3303/files#r942509480, if the user does not have read access we should put in some screen reader text explaining that the action menu is not available due to their access level. Right now it reads as Blank in this case and does not render very nicely.

![image](https://user-images.githubusercontent.com/484484/183927511-810b30c6-52a5-49c2-ad0a-99e1cec04f4b.png)

Reply:
I don't have a workspace that has 'No Access' as its access level, is there a way I can check if my code works?

Reply:
@yunip20 if you log in as "hermione.owner@test.firecloud.org", you will see a number of workspaces in this state (sort by access level).